Prototyper/Classes/Feedback/EditScreenshotView.swift

```diff
@@ -97,17 +97,21 @@ struct EditScreenshotView: View {
     }
     
     private func save() {
-        self.drawings = [Drawing]()
-        self.model.screenshot = UIApplication.shared.windows.first?.asImage(rect: rect) ?? UIImage()
+        self.model.markupDrawings.removeAll()
+        self.allDrawings.forEach({ self.model.markupDrawings.append($0) })
```

Comment:
I would suggest to replace 100 and 101 with:
```suggestion
        self.model.markupDrawings = allDrawings
```